src/pudl/transform/eia861.py

```diff
@@ -1579,27 +1595,41 @@ def operational_data(tfr_dfs):
     transformed_od = (
         _clean_nerc(raw_od, idx_cols)
         .assign(
-            revenue_from_credits_or_adjustments=lambda x: x.revenue_from_credits_or_adjustments * 1000.0,
-            revenue_from_delivery_customers=lambda x: x.revenue_from_delivery_customers * 1000.0,
-            revenue_from_other=lambda x: x.revenue_from_other * 1000.0,
-            revenue_from_retail_sales=lambda x: x.revenue_from_retail_sales * 1000.0,
-            revenue_from_sales_for_resale=lambda x: x.revenue_from_sales_for_resale * 1000.0,
-            revenue_from_transmission=lambda x: x.revenue_from_transmission * 1000.0,
-            revenue_total=lambda x: x.revenue_total * 1000.0,
+            credits_or_adjustments_revenue=lambda x: x.credits_or_adjustments_revenue * 1000.0,
+            delivery_customers_revenue=lambda x: x.delivery_customers_revenue * 1000.0,
+            other_revenue=lambda x: x.other_revenue * 1000.0,
+            retail_sales_revenue=lambda x: x.retail_sales_revenue * 1000.0,
+            sales_for_resale_revenue=lambda x: x.sales_for_resale_revenue * 1000.0,
+            transmission_revenue=lambda x: x.transmission_revenue * 1000.0,
+            total_revenue=lambda x: x.total_revenue * 1000.0,
```

Comment:
If all of these columns will be squished in one via `_tidy_class_dfs`, could we run this assign on just one column once these are squished?

Reply:
That's a great point!